modules/python/src2/cv2.cpp

```diff
@@ -411,12 +411,84 @@ PyObject* pyopencv_from(const Mat& m)
     return o;
 }
 
-template<typename _Tp, int m, int n>
-PyObject* pyopencv_from(const Matx<_Tp, m, n>& matx)
-{
-    return pyopencv_from(Mat(matx));
+#define PYOBJ_FROM(_Tp) \
+static PyObject* pyopencv_from(const _Tp& src) \
+{ \
+    return pyopencv_from(Mat(src)); \
 }
 
+#define PYOBJ_TO(_Tp) \
+static bool pyopencv_to(PyObject* obj, _Tp& dst, const ArgInfo info) \
+{ \
+    Mat src; \
+    Size dstSize = Mat(_Tp()).size(); \
+ \
+    if (!pyopencv_to(obj, src, info)) \
+        return false; \
```

Comment:
BTW, this call returns `cv::Mat` with double type (CV_64F) very often.
This works for these floating-point objects, but the name of macro should be changed to prevent misusing of this.